conans/model/manifest.py

```diff
@@ -89,22 +88,14 @@ def load(folder):
         return FileTreeManifest.loads(text)
 
     def __repr__(self):
+        # Used for serialization and saving it to disk
         ret = ["%s" % self.time]
```

Comment:
Not sure if it is important, but I think this should be UTC format to be the same as in the old `__str__()` method

Reply:
Not sure what you mean, we cannot change this, this is the way the conanmanifest.txt files are stored on disk, it is an integer (seconds), always UTC, so this cannot be changed, it could break a lot of things.

Reply:
I have restored the ``__str__`` function, it was a very corner case for displaying a message when Uploading and the manifest in the server is newer.